### PR TITLE
Creating case study

### PR DESCRIPTION
**Pull request by PatrykHub**

### CODE REVIEW

**Review by agak33 — CHANGES_REQUESTED** [review]
pls change the file name to something more descriptive and fix the part where an exception is raised

**Review by agak33 — APPROVED** [review]

File: mcda/electre/credibility.py [hunk 1]
```diff
@@ -58,16 +53,18 @@ def credibility_comprehensive(
...
-        or comprehensive discordance :math:`D(a, b)`
+    :param discordance: discordance :math:`D` or non-discordance :math:`{\\Delta}`
+
     :param is_non_discordance: ``True`` if non-discordance is provided, otherwise ``False``,
         defaults to ``True``
 
     :return: credibility table :math:`S` with values from [0, 1] interval
     """
+    if not is_non_discordance:
+        discordance = 1 - discordance
```
Comment (agak33) [inline mcda/electre/credibility.py:67]: Haven't we talked that such things should be AFTER THE VALIDATION?

File: mcda/electre/outranking/crisp_outranking.py [hunk 2]
```diff
@@ -32,48 +32,47 @@ class OutrankingRelation(Enum):
...
-    return credibility >= cutting_level
+    _check_index_value_interval(value, "value")
+    _check_index_value_interval(cutting_level, "cutting level", minimal_val=0)
+    return value >= cutting_level
 
 
-def crisp_outranking_cut(
-    credibility_table: pd.DataFrame,
+def crisp_cut(
+    table: pd.DataFrame,
```
Comment (agak33) [inline mcda/electre/outranking/crisp_outranking.py:53]: I'd prefer a little bit more descriptive name than just `table`. Maybe something like `index_table`? :3

**Review by a-pralat — CHANGES_REQUESTED** [review]
Please make changes from Aga's comments

File: mcda/electre/outranking/crisp_outranking.py [hunk 3]
```diff
@@ -32,48 +32,47 @@ class OutrankingRelation(Enum):
...
+    _consistent_df_indexing(indices_table=indices_table)
     return pd.DataFrame(
         [
             [
-                crisp_outranking_cut_marginal(
-                    credibility_table.loc[alt_name_a][alt_name_b], cutting_level
-                )
-                for alt_name_b in credibility_table.columns.values
+                crisp_cut_marginal(indices_table.loc[alt_name_a][alt_name_b], cutting_level)
+                for alt_name_b in indices_table.index.values
```
Comment (agak33) [inline mcda/electre/outranking/crisp_outranking.py:70]:
```suggestion
                for alt_name_b in indices_table.columns.values
```

**Review by agak33 — CHANGES_REQUESTED** [review]

File: mcda/electre/outranking/crisp_outranking.py [hunk 4]
```diff
@@ -32,48 +32,47 @@ class OutrankingRelation(Enum):
...
             ]
-            for alt_name_a in credibility_table.index.values
+            for alt_name_a in indices_table.index.values
         ],
-        index=credibility_table.index,
-        columns=credibility_table.columns,
+        index=indices_table.index,
+        columns=indices_table.index,
```
Comment (agak33) [inline mcda/electre/outranking/crisp_outranking.py:75]:
```suggestion
        columns=indices_table.columns,
```

**Review by a-pralat — APPROVED** [review]

**Pull request merged**